### PR TITLE
blog: new blog post 'urbit for communities'

### PR DESCRIPTION
**Pull request by urcades**
Posting up the draft

### CODE REVIEW

**Comment by galenwp** [comment]
lgtm

**Comment by galenwp** [comment]
oh wait, there's a link missing. i'll add

File: content/blog/urbit-is-for-communities.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
++++
+title = "Urbit is for Communities"
+date = 2020-03-23
+description = "Urbit is for giving communities the tools to evolve their own environments; for us all to feel a sense of life and self-directedness in the digital world."
```
Comment (matildepark) [inline content/blog/urbit-is-for-communities.md]: The exact wording I liked was "shape their own environments" — evolve seems not aggressive enough. idk, I'm being pedantic, clearly. @galenwp

File: content/blog/urbit-is-for-communities.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+Urbit provides an experience with less distance between ordinary use and total customization. To start, we’ve created Ikea-like, Lego-like and woodshop-like levels of usability. What we’ve built isn’t nearly as complex as the physical world of course, but it’s a start.
+
+For most people, customizing your Urbit should be a matter of picking from an available suite of modules. Not unlike going to Ikea. For some people, recombining interfaces and data sources within Ubit should feel like playing with Legos. And for others, who want to dive in and write code, it’s the same as going into a shop full of high quality tools.
+
+We’re not quite to this level of sophistication, but we’re on the path. Let’s talk about where we are and where we’re going.
+
+
+## OS 1
+
+Our roughest attempt to satisfy this goal is hovering on the horizon. We call it OS 1.
```
Comment (matildepark) [inline content/blog/urbit-is-for-communities.md]: Inadvertently a self-own by saying it's our roughest ever release; maybe "The foundations for satisfying this goal are hovering on the horizon" @galenwp

**Review by matildepark — APPROVED** [review]
Besides some copywriting feedback looks technically good

File: content/blog/urbit-is-for-communities.md (hunk 2 above)
Comment (urcades) [reply content/blog/urbit-is-for-communities.md]: I can see where you're coming from where 'rough' is translated as a bad/negative word, but I read 'rough' in the sense of 'rough draft', which is basically a positive thing if you believe in sharing rough drafts with people

Comment (galenwp) [reply content/blog/urbit-is-for-communities.md]: I'd be fine with roughest -> rough

File: content/blog/urbit-is-for-communities.md (hunk 1 above)
Comment (galenwp) [reply content/blog/urbit-is-for-communities.md]: I'm fine with switching to 'shape'. Although if you read the piece, the argument has to do not just with shaping, but shaping over time.

**Pull request merged**